docs/getting-started/security-ui.asciidoc

```diff
@@ -51,6 +51,20 @@ View event and host counts specific to Elastic data shippers and apps, such as *
 [role="screenshot"]
 image::images/events-count.png[]
 
+*Threat Intelligence card*
```

Comment:
`card` is a new term that I haven't seen used elsewhere; I think we'd been referring to it as such during development, so I think that might be how it found its way here. The data conveyed here is _very_ similar to the "host and network events" views; I think the title here could simply be "Threat Intelligence": 
```suggestion
*Threat Intelligence*
```